wp-content/plugins/core/src/Blocks/Middleware/Post_Loop/README.md

```diff
@@ -0,0 +1 @@
+# Post Loop Middleware
```

Comment:
Is there more you were wanting to add here?

Reply:
Removed this file.